js/main.js

```diff
@@ -80,20 +89,47 @@ function setTab(selectedTab) {
     }
     tabButtons[selectedTab].classList.add('active');
 
-    Dom.get().byId('content').animate(
-        [
-            {offset: 0.0, backgroundSize: '0 100%'},
-            {offset: 0.1, backgroundSize: '24px 100%'},
-            {offset: 0.6, backgroundSize: '24px 100%'},
-            {offset: 1.0, backgroundSize: '0 100%'},
-        ],
-        {
-            duration: 600,
-            easing: 'ease-in-out',
-            iterations: 1,
-        },
-    );
-
+    const content = Dom.get().byId('content');
+    const animBar = content.querySelector('.bar-animation');
+
+    if(isVerticalBarAnimation) {
+        content.style.borderTop = 'none';
+        content.style.borderLeft = '3px solid var(--active-color)';
+        animBar.className = 'bar-animation vertical';
```

Comment:
```suggestion
        animBar.classList.add('vertical');
        animBar.classList.remove('horizontal');
```
ist netter als das gesamte `class` attribute neu zu schreiben.

Kannst du entsprechend auch bei horizontal anwenden.